### ISSUE TITLE
PNG template file not being uploaded 

### ISSUE THREAD

**Issue by LaurenaRehbein** [moovida/remarkable-hyutilities#2]
**My Goal:**

To upload a new PNG document as a template, using the "templates" mode. 

**Steps:**

1. Place PNG file in local dir
2. Open Rermarkable HyUtilities and connect
3. Select `templates`
4. Refresh remote
5. Click `Upload Local`

**What I expected:** 

I expected the `PNG` file to transfer to the tablet, and the `templates.json` file to update. 

**What happened instead:** 

The `templates.json` file was updated but the `PNG` file did not transfer. There was an error message (screenshot below) and the file did not appear in the remote dir. 

**Error:** 

![http://cld.wthms.co/x9MXQH](http://cld.wthms.co/x9MXQH+)
Full Image: http://cld.wthms.co/x9MXQH+ 

All other modes are working well! Looking forward to a possible solution here.

**Comment by moovida** [comment]
hi Lauren, I am on vacation and have no way to check this. I will sue have a look at this issue and fix it if I am able to reproduce this, but it will take some time.
Thanks a lot for reporting this.

**Comment by liesbethjj** [comment]
Hi, I tried uploading templates from windows as well as mac. They are uploaded, but those uploaded from windows I cannot use (I can choose them from the settings>choose template menu, but they are invsible in the document. the background stays blank). When I tried to upload the same templates (but whith chaged names, so I could see the difference) from the mac, they uploaded but the names are not visible in the list of possible templates to choose from, all I see is a big questionmark, So when I choose one of those questionmarks at random, it turns out to be one of my own templates. 
best wishes, 
Liesbeth
